appimage/Readme.md

```diff
@@ -0,0 +1,24 @@
+AppImage Packaging
+==================
+
+Creates an AppImage that contains `pktvisord` `pktvisor-pcap` and `pktvisor-cli`. This implementation builds the AppImage in a docker using the assets generated by the docker [build](../docker). Because not all of the assets are statically complied we use [Exodus](https://github.com/intoli/exodus) to capture a minimal set of decencies for the AppImage 
+
+
+## Build:
+`make pktvisor-x86_64.AppImage`
+
+## Development:
+Because the build can take a while you may want to build the appimage from the latest docker image on docker hub. To do this you can set the `DEV_MODE=` environment to anything. Like so:
+
+`DEV_MODE=t make pktvisor-x86_64.AppImage`
+
+
+## Usage:
+To use the AppImage from the command line you can specify the binary you want to run as the first argument following the pattern:
+
+`
+./pktvisor-x86_64.AppImage [pktvisord|pktvisor-pcap|pktvisor-cli] <args>
+`
+
+If you don't specify all args are passed to `pktvisord` by default.
```

Comment:
To be clear, this means you can leave off specifying the binary, just pass args, and in that case it calls pktvisord with the args? so

"If you don't specify a binary, all args are passed to `pktvisord` by default." ?